domain-ee/ee-ep-merge-app/src/python_src/model/merge_job.py

```diff
@@ -4,17 +4,18 @@
 
 from pydantic import BaseModel, conint
 
-JobState = Enum('JobState', [
-    'COMPLETED_ERROR',
-    'PENDING',
-    'RUNNING_GET_PENDING_CLAIM',
-    'RUNNING_GET_PENDING_CLAIM_CONTENTIONS',
-    'RUNNING_GET_EP400_CLAIM_CONTENTIONS',
-    'RUNNING_SET_TEMP_STATION_OF_JURISDICTION',
-    'RUNNING_MERGE_CONTENTIONS',
-    'RUNNING_UPDATE_PENDING_CLAIM_CONTENTIONS',
-    'RUNNING_CANCEL_EP400_CLAIM',
-    'COMPLETED_SUCCESS'])
+
+class JobState(str, Enum):
+    COMPLETED_ERROR = 'COMPLETED_ERROR'
+    PENDING = 'PENDING'
+    RUNNING_GET_PENDING_CLAIM = 'RUNNING_GET_PENDING_CLAIM'
+    RUNNING_GET_PENDING_CLAIM_CONTENTIONS = 'RUNNING_GET_PENDING_CLAIM_CONTENTIONS'
+    RUNNING_GET_EP400_CLAIM_CONTENTIONS = 'RUNNING_GET_EP400_CLAIM_CONTENTIONS'
+    RUNNING_SET_TEMP_STATION_OF_JURISDICTION = 'RUNNING_SET_TEMP_STATION_OF_JURISDICTION'
+    RUNNING_MERGE_CONTENTIONS = 'RUNNING_MERGE_CONTENTIONS'
+    RUNNING_MOVE_CONTENTIONS_TO_PENDING_CLAIM = 'RUNNING_MOVE_CONTENTIONS_TO_PENDING_CLAIM'
+    RUNNING_CANCEL_EP400_CLAIM = 'RUNNING_CANCEL_EP400_CLAIM'
+    COMPLETED_SUCCESS = 'COMPLETED_SUCCESS'
```

Comment:
What's the rationale behind this change? The new version is a lot more verbose.

Reply:
Essentially,  using the strings instead makes the response more readable. i.e., `"state":"COMPLETED_SUCCESS"` instead of `"state":9`

Reply:
That seems like the presentation layer's responsibility to display job states informatively. But there are always little hacks to make it work on the data object's side; like, of the top of my head, you could patch the bare-bones enum type with

```python
JobState.__repr__ = JobState.__str__ = lambda self: self.name
```

Then again, it may not be worth the effort for this relatively self-contained case of repetitiveness. Your call.